eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentPacket.scala

```diff
@@ -124,7 +121,6 @@ object IncomingPaymentPacket {
               case DecodedEncryptedRecipientData(blindedPayload, nextBlinding) =>
                 validateBlindedChannelRelayPayload(add, payload, blindedPayload, nextBlinding, nextPacket)
             }
-          case None if add.blinding_opt.isDefined => Left(InvalidOnionPayload(UInt64(12), 0))
```

Comment:
You're right, I removed it because when we reach that case, I wanted to have the same behavior as a node that doesn't support route blinding (and thus doesn't understand this blinding point), but since we use tlv tag 0 in `update_add_htlc`, such a node would reject the message. I'll fix that.

Reply:
Added in https://github.com/ACINQ/eclair/pull/2408/commits/f83d5de31bebe5aa0a1b195d9856ecb98ba2a9e4